test/evergreen/import_compatibility_test.sh

```diff
@@ -38,7 +47,7 @@ create_file()
     echo "Branch \"$1\" creating and populating \"$2\""
     echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
 
-    wt_cmd="$1/wt"
+    wt_cmd="$1/build/wt"
     test_dir="$1/WT_TEST/"
```

Comment:
I think we have a convention of having the `WT_TEST` directory created under the build directory. Not strong opinion just list here for consideration.

Reply:
Fair point. I've updated it to be under the build directory: 5a2c1a343d8f86b328d000170573c713aa43af94